src/styles/imageslider.css

```diff
@@ -0,0 +1,17 @@
+.trans {
+    opacity: 0;
+}
+
+.skeltrans {
+    opacity: 0;
+    
+}
+
+.skelnorm {
+    opacity: 1;
+    transform: translateX(-10rem);
```

Comment:
When the skeleton is being rendered instead of the actual image, the skeleton moves to the left side for a second right before the image is rendered. Possibly this is the issue??